Package ocaml-varint.1.0

XnuKernPoll opened the pull request:
### `ocaml-varint.1.0`

A simple VarInt implementation.



---
* Homepage: https://github.com/XnuKernPoll/ocaml-varint
* Source repo: http://github.com/XnuKernPoll/ocaml-varint.git
* Bug tracker: https://github.com/XnuKernPoll/ocaml-varint/issues

---
### opam-lint failures
- **WARNING** 97 long description unspecified

---

:camel: Pull-request generated by opam-publish v0.3.5

camelus commented:
##### :white_check_mark: All lint checks passed <small>0b0d9a8344708508ea05817e28ba95c855ae7e51</small>

* These packages passed lint tests: `ocaml-varint.1.0`


---


##### :white_check_mark: Installability check (9136 &rarr; 9137)

* new installable packages (1): `ocaml-varint.1.0`

pmetzger commented on packages/ocaml-varint/ocaml-varint.1.0/descr:
```diff
@@ -0,0 +1 @@
+A simple VarInt implementation.
```
I note there is no \n at the end of this line.

pmetzger commented:
Just an aside: when I see interesting package names go by, I often try to look at the package and see if it is interesting to me. The entire documentation (including in your README and your mli files) for this package is

> A simple VarInt implementation.

which makes it hard for potential users to figure out that they might want to use your code.

hannesm commented:
This may be not well documented (it is in https://github.com/ocaml/opam-repository/wiki/PR-checks), in opam-repository we try to avoid `ocaml-` prefixes and `-ocaml` suffixes for package names.  Would you be so kind to rename your package to `varint` (and I agree with @pmetzger that varint should be a bit better documented).  Thanks for your contribution!

XnuKernPoll commented:
Alright, I'll fix the package descr, change the name, and add a tutorial in the read me, and add some comments in the mli files.

XnuKernPoll commented:
I added the changes you guys gave me, fixed the api, and added Length Prefixing
https://github.com/ocaml/opam-repository/pull/12377

Out of curiosity do append, concat, or sub copy, because I know that bigarray can use layout transformation to avoid copies.
